src/shared/services/document-management/document-management.service.ts

```diff
@@ -5,13 +5,16 @@ import { HttpService } from '../http';
 import { Headers } from '@angular/http';
 import { AbstractAppConfig } from '../../../app.config';
 import { map } from 'rxjs/operators';
+import { delay } from 'rxjs/internal/operators';
 
 @Injectable()
 export class DocumentManagementService {
   private static readonly HEADER_ACCEPT = 'Accept';
   private static readonly HEADER_CONTENT_TYPE = 'Content-Type';
   private static readonly PDF = 'pdf';
   private static readonly IMAGE = 'image';
+  // This delay has been added in order to be able to display any temporally status messages in the GUI component that uses this service.
```

Comment:
can you change to:
`This delay has been added to give some time to the user to see the info messages on the document upload field for cases when uploads are very fast`